test/cpp/jit/test_gpu.cpp

```diff
@@ -4496,9 +4496,6 @@ void testGPU_FusionCacheBcast() {
   tv8->axis(-1)->parallelize(ParallelType::TIDx);
 
   constexpr int M = 92, N = 500;
```

Comment:
keep an empty line below the constexp line (the constants are used in more places than just the immediate following "paragraph", so we should not hint that it's part of it)